keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -537,15 +537,6 @@ class ClusterHandler(
           0 -> emptyList()
           else -> listOf(startingRefId.toString())
         },
```

Comment:
probably also needs to be changed in the `TitusClusterHandler`

Reply:
I was hoping to leave that for later, but will do.